### DISCUSSION TITLE
科技戰爭》烏克蘭使用臉部識別AI技術、Google表單，協助尋找俄國陣亡軍人身份

### DISCUSSION THREAD

**Discussion by Yuhsin000** [1101-GS4719-C/Python-Community-GS4719-C#535] [General]
### 此次烏俄戰爭除了有使用 AI 作戰外，也有使用AI來達到人道的目的——協助尋找遺體。
(先前有同學提到作戰的部分 : https://github.com/NCU-GS4719-Python/Python-Community/discussions/506 )
(此篇討論AI人道應用 :  https://tw.news.yahoo.com/%E7%A7%91%E6%8A%80%E8%88%87%E6%88%B0%E7%88%AD-%E7%83%8F%E5%85%8B%E8%98%AD%E4%BD%BF%E7%94%A8%E8%87%89%E9%83%A8%E8%AD%98%E5%88%A5ai%E6%8A%80%E8%A1%93-google%E8%A1%A8%E5%96%AE-%E5%8D%94%E5%8A%A9%E5%B0%8B%E6%89%BE%E4%BF%84%E5%9C%8B%E9%99%A3%E4%BA%A1%E8%BB%8D%E4%BA%BA%E8%BA%AB%E4%BB%BD-124001044.html)

### Clearview 會從公共網絡（包括社交媒體）抓取照片進行比對，以找出目標人物的身分。
雖然有侵犯隱私權的疑慮，但我認為在戰爭狀況下有時顧不了那麼多，大家對此較為寬容，但也不禁讓人思考，一時的寬容是否會造成日後資安的問題，不知道大家對這個看法如何 ?

### 除了Clearview 外，還有google表單輔助
俄羅斯軍人親屬可以上去填寫，要求烏克蘭軍方回收某位死去俄國士兵的遺體。

### 為何有些政府不使用這項技術來協助辨識遺體?
1. 認錯人 : 造成不必要的悲傷
2. 不可靠 : 殘破不堪和沒有表情的五官可能會使死者的臉部識別變得不可靠
3. 無可避免地會蒐集到戰場傷亡的照片 : 沒有針對這種技術濫用的保護措施，會被擴大使用

### 結論與感想
AI 在此次戰爭中扮演了重要的角色，這兩年來戰爭、疫情促使科技大幅進步，然而相關措施(如法規、共識等)並未跟上，人類在科技飛快進步的同時也須花時間好好討論相應的問題與對策。

**Comment by Sazmafuji** [comment]
此技術雖然是有關於戰爭，但希望可以透過此技術讓戰爭停止

**Comment by ThomasMeng0515** [comment]
如果傷亡士兵的照片沒有妥善保管與處理，意外流出的話也會成為很嚴重的隱私問題

**Comment by sean1325418** [comment]
科技發展最快的時候往往都是在戰爭時期

**Comment by PoooYen** [comment]
沒有想到科技是以這樣遺憾的方式來增進人類福祉.....

**Comment by HuanYuuuuuu** [comment]
人類在科技飛快進步的同時也要注意相關規範來限制技術的濫用

**Comment by cordydeng** [comment]
這個技術需要用到大量的圖片資料，會不會有侵犯隱私權的疑慮?
